adagrams/game.py

```diff
@@ -1,11 +1,104 @@
+from string import ascii_uppercase
+from random import randint , choice
+
+
+
 def draw_letters():
-    pass
+    letter_pool = {
+        'A': 9, 
+        'B': 2, 
+        'C': 2, 
+        'D': 4, 
+        'E': 12, 
+        'F': 2, 
+        'G': 3, 
+        'H': 2, 
+        'I': 9, 
+        'J': 1, 
+        'K': 1, 
+        'L': 4, 
+        'M': 2, 
+        'N': 6, 
+        'O': 8, 
+        'P': 2, 
+        'Q': 1, 
+        'R': 6, 
+        'S': 4, 
+        'T': 6, 
+        'U': 4, 
+        'V': 2, 
+        'W': 2, 
+        'X': 1, 
+        'Y': 2, 
+        'Z': 1
+    }
+    hand = []
+
+
+    while len(hand) < 10:
+        pick = choice(ascii_uppercase)
+        if letter_pool[pick] > 0:
+            hand.append(pick)
+            letter_pool[pick] -= 1
+    return hand
+
+    # while len(hand) < 10:
+    #     pick = ascii_uppercase[randint(0, len(ascii_uppercase)-1)]
+    #     if letter_pool[pick] > 0:
+    #         hand.append(pick)
+    #         letter_pool[pick] -= 1
+    # return hand
+
 
 def uses_available_letters(word, letter_bank):
-    pass
+    word_list = list(word.upper())
+    letter_bank_list = list(letter_bank)
+
+    for letter in word_list:
```

Comment:
Instead of converting `word` into a list, you can directly iterate over word like:

```python
letter_bank_list = list(letter_bank)

for letter in word.upper():
    # rest of your code
```

Reply:
Thank you, I'll fix that

Reply:
Done